### ISSUE TITLE
Bug when reading sz754 incorrect length

### ISSUE THREAD

**Issue by Nauja** [FHPythonUtils/BinaryIOTools#2]
### System info

- OS: Windows
- Version: Python 3.7+

### Describe the bug

The _sz754 method doesn't correctly read an sz754 string.

sz754 string format described in documentation is:

```
Note, string format is:
	uint32   n+1  Number of bytes that follow, including the zero byte
	byte[n]  ...  String data in Unicode, encoded using UTF-8
	byte     0    Zero marks the end of the string.
```

Implementation is:

```python
nchars = self.u32
if nchars == 0:
    return ''
data = self.data[self.index:self.index + nchars - 1]
self.index += nchars
```

### Expected outcome

_sz754 should read 4 bytes + string size bytes + 1 extra byte for 0 at the end of the string

### Actual outcome

_sz754 read 4 bytes + (string size - 1) bytes

This results in:

```python
> io = binaryiotools.IO()
> io.sz754 = "hello"
> io.sz754
hell
> io.index
9
```

Note the index is wrong too as it should take into account the extra 0 character

**Comment by FredHappyface** [comment]
Thanks for the report, I shall take a look at this :)

**Comment by FredHappyface** [comment]
Fixed in #4
